activemq_xml/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - activemq_xml
 
+## 1.8.2 / 2020-08-12
+
+* [Fixed] Clean http config whitespaces. See [#7339](https://github.com/DataDog/integrations-core/pull/7339).
```

Comment:
Should this PR be moved to a `changelog/no-changelog`? It seems to only be removing trailing white spaces in example `conf.yaml` to please our linters. I'm not sure it's worth adding in the changelog (and including in a release, these updated files being available publicly in master anyway) - feels very noisy?